yoda/src/main/java/life/jukebox/HandshakeClient.java

```diff
@@ -0,0 +1,69 @@
+package life.jukebox.yoda;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A simple client that requests a greeting from the {@link HelloWorldServer}.
+ */
+public class HandshakeClient {
+  private static final Logger logger = Logger.getLogger(HandshakeClient.class.getName());
+
+  private final ManagedChannel channel;
+  private final YodaGrpc.YodaBlockingStub blockingStub;
+
+  /** Construct client connecting to HelloWorld server at {@code host:port}. */
+  public HandshakeClient(String host, int port) {
+    this(ManagedChannelBuilder.forAddress(host, port)
+        // Channels are secure by default (via SSL/TLS). For the example we disable TLS to avoid
+        // needing certificates.
+        .usePlaintext(true)
+        .build());
+  }
+
+  /** Construct client for accessing RouteGuide server using the existing channel. */
+  HandshakeClient(ManagedChannel channel) {
+    this.channel = channel;
+    blockingStub = YodaGrpc.newBlockingStub(channel);
+  }
+
+  public void shutdown() throws InterruptedException {
+    channel.shutdown().awaitTermination(5, TimeUnit.SECONDS);
+  }
+
+  /** Say hello to server. */
+  public void shake(String name) {
+    logger.info("Will try to greet " + name + " ...");
+    HandshakeRequest request = HandshakeRequest.newBuilder().setName(name).build();
+    HandshakeResponse response;
+    try {
+      response = blockingStub.shake(request);
+    } catch (StatusRuntimeException e) {
+      logger.log(Level.WARNING, "RPC failed: {0}", e.getStatus());
+      return;
+    }
+    logger.info("Greeting: " + response.getMessage());
+  }
+
+  /**
+   * Yoda server. If provided, the first element of {@code args} is the name to use in the
+   * greeting.
+   */
+  public static void main(String[] args) throws Exception {
+    HandshakeClient client = new HandshakeClient("localhost", 37000);
```

Comment:
Both of these vars (host and port) should be able to be configured using environment variables. Your code needs to check environment variables (JB_YODA_SERVICE_HOST, JB_YODA_SERVICE_PORT) - if they don't exist, you should fallback to localhost and 37000.
For example look at [FE's app client](https://github.com/PedramPejman/jukebox/blob/master/fe/src/app_client.js)

Reply:
Also, you should never have literals/constants inside your code. So at the top of the file, define the two constants (DEFAULT_JB_YODA_SERVICE_HOST and DEFAULT_JB_YODA_SERVICE_PORT)